web/template/course-overview.gohtml

```diff
@@ -101,11 +101,20 @@
                 <div class="flex justify-between h-16 px-3"
                      :class="showPlanned && 'border-b dark:border-gray-800'">
                     <span class="bg-info rounded-full text-lg font-bold text-white px-5 my-auto">Upcoming</span>
-                    <button class="hover:bg-gray-200 dark:hover:bg-gray-600 rounded px-2 my-auto"
-                            @click="showPlanned = !showPlanned;">
-                        <span class="text-sm font-semibold uppercase dark:text-white"
-                              x-text="showPlanned ? '&#x25B2; hide' : '&#x25BC; show'">&#x25B2; hide</span>
-                    </button>
+                    <div class="my-auto">
+                        <button class="hover:bg-gray-200 dark:hover:bg-gray-600 rounded px-2 my-auto"
+                                @click="showPlanned = !showPlanned;">
+                            <span class="text-sm font-semibold uppercase dark:text-white"
+                                  x-text="showPlanned ? '&#x25B2; hide' : '&#x25BC; show'">&#x25B2; hide</span>
+                        </button>
+                        <button class="hover:bg-gray-200 dark:hover:bg-gray-600 rounded px-2"
```

Comment:
An `<a>` tag might make more sense, or does that break the UI?

Reply:
I assumed it would, but it doesn't actually. I've changed it.